docs/getting_started/installation.md

```diff
@@ -13,83 +13,89 @@ Please make sure your system meets the following specs:
-```shell
-curl -LJO https://raw.githubusercontent.com/StanfordVL/OmniGibson/main/docker/run_docker.sh
-curl -LJO https://raw.githubusercontent.com/StanfordVL/OmniGibson/main/docker/run_docker_gui.sh
-chmod a+x run_docker.sh
-chmod a+x run_docker_gui.sh
-```
-
-??? question annotate "What is being installed?"
-
-    Our docker image automatically ships with a pre-configured conda virtual environment named `omnigibson` with Isaac Sim and **`OmniGibson`** pre-installed. Upon running the first time, our scene and object assets will automatically be downloaded as well. (1)
+    ??? question annotate "What is being installed?"
 
-1.  📊 **Worried about dataset size?** We will ask whether you want to install our small demo dataset or full dataset of assets!
+        Our docker image automatically ships with a pre-configured conda virtual environment named `omnigibson` with Isaac Sim and **`OmniGibson`** pre-installed. Upon running the first time, our scene and object assets will automatically be downloaded as well. (1)
 
+    1.  📊 **Worried about dataset size?** We will ask whether you want to install our small demo dataset or full dataset of assets!
 
-Then, simply launch the desired script:
+    Then, simply launch the shell script:
 
-=== "Headless"
+    === "Headless"
 
-    ```{.shell .annotate}
-    sudo ./run_docker.sh <DATA_PATH> # (1)!
-    ```
-
-    1.  `<DATA_PATH>` specifies where data will be stored on your machine (if no `<DATA_PATH>` is specified, it defaults to `./omnigibson_data`). This needs to be called each time the docker container is run!
+        ```{.shell .annotate}
+        sudo ./run_docker.sh -h <ABS_DATA_PATH> # (1)!
+        ```
 
+        1.  `<ABS_DATA_PATH>` specifies the **absolute** path data will be stored on your machine (if no `<ABS_DATA_PATH>` is specified, it defaults to `./omnigibson_data`). This needs to be called each time the docker container is run!
 
-=== "GUI"
 
-    ```{.shell .annotate}
-    sudo ./run_docker_gui.sh <DATA_PATH> # (1)!
-    ```
+    === "GUI"
 
-    1.  `<DATA_PATH>` specifies where data will be stored on your machine (if no `<DATA_PATH>` is specified, it defaults to `./omnigibson_data`). This needs to be called each time the docker container is run!
+        ```{.shell .annotate}
+        sudo ./run_docker.sh <ABS_DATA_PATH> # (1)!
+        ```
 
-??? warning annotate "Are you using NFS or AFS?"
+        1.  `<ABS_DATA_PATH>` specifies the **absolute** path data will be stored on your machine (if no `<ABS_DATA_PATH>` is specified, it defaults to `./omnigibson_data`). This needs to be called each time the docker container is run!
 
-    Docker containers are unable to access NFS or AFS drives, so if `run_docker.sh` or `run_docker_gui.sh` are located on an NFS / AFS partition, please set `<DATA_PATH>` to an alternative data directory located on a non-NFS / AFS partition.
+    The `-h --headless` argument is used to switch between gui and headless mode.
```

Comment:
I think we can remove this line. If anything, add something like `-h / --headless` is used to run in headless mode in the 1. annotation

Reply:
Sure, I'll just remove this line